app/controllers/receipt_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ReceiptController < ApplicationController
+  def index
+    @name = params[:name]
+    @price = params[:price]
```

Comment:
A recept typically doesn't have a single price on it.  A receipt typically has a total, and the items on the receipt have prices.